drivers/unix/os_unix.cpp

```diff
@@ -845,4 +845,7 @@ OS_Unix::OS_Unix() {
 	_set_logger(memnew(CompositeLogger(loggers)));
 }
 
+OS_Unix::~OS_Unix() {
+}
```

Comment:
This still needs to be addressed. There is no reason to introduce this change in this PR.